debug/Dockerfile

```
@@ -0,0 +1,36 @@
+FROM golang:1.15
+LABEL maintainer="NebulousLabs <devs@nebulous.tech>"
+
+ENV GOOS linux
+ENV GOARCH amd64
+
+WORKDIR /root
+
+RUN apt-get update && \
+    apt-get -y dist-upgrade && \
+    apt-get -y autoremove && \
+    apt-get -y install python3-pip sudo wget curl zip vim nano nload psmisc less mime-support && \
+    apt-get clean && \
+    wget -O- -nv https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.30.0 && \
+    pip3 install codespell && \
+    go get -d -u gitlab.com/NebulousLabs/Sia/... && \
+    go get gitlab.com/NebulousLabs/analyze && \
+    ln -s $GOPATH/src/gitlab.com/NebulousLabs/Sia ./ && \
+    cd $GOPATH/src/gitlab.com/NebulousLabs/Sia && \
+    make release && \
+    cd
+
+# Setup the actual Sia environment.
+ARG SIA_DIR="/sia"
+ARG SIA_DATA_DIR="/sia-data"
+ARG SIAD_DATA_DIR="/sia-data"
+
+COPY ./debug/.vimrc ./debug/.bashrc scripts/*.sh ./
```

Would something like this work?

```suggestion
COPY ./debug/ scripts/*.sh ./
```

---

Yes but that would copy the Dockerfile as well. That would be more confusing than useful. I can simplify it to this
```dockerfile
COPY ./debug/.*rc scripts/*.sh ./
```

---

Ah okay. I prefer the first version in that case -- explicitly listing the files seems better than relying on a commonality in the names.